test/board/devices/output/7seg.test.ts

```diff
@@ -0,0 +1,65 @@
+import { Byte, Halfword, Word } from 'types/binary'
+import { SEVENseg } from '../../../../src/board/devices/output/7seg'
```

Comment:
Maybe you could also add a test to see what happens if a Word is written to the binary interface at address `0x0x60000115` because this should cut off the rest.